spec/support/seeds.rb

```diff
@@ -36,5 +36,14 @@ def seeds
         client_data_json: 'eyJ0eXBlIjoid2ViYXV0aG4uY3JlYXRlIiwiY2hhbGxlbmdlIjoiY0lDOUFKclpEZ3FhcllrTWtqN3NZNUU4Wlg0VVpjSmI1SzhDUzZLMnc0byIsIm9yaWdpbiI6Imh0dHA6XC9cL2xvY2FsaG9zdDozMDAwIiwiYW5kcm9pZFBhY2thZ2VOYW1lIjoiY29tLmNocm9tZS5jYW5hcnkifQ==',
       },
     },
+    android_key_direct: {
+      credential_creation_options: {
+        challenge: 'v2h1c2VybmFtZXQ0aV9ObTRobjFDeEkwSGc3OHhzTWljaGFsbGVuZ2VQtubEDC4OSpGHebHLLNerFf8'
+      },
+      authenticator_attestation_response: {
```

Comment:
Generated by https://fidoalliance.org/certification/functional-certification/conformance/ and these certificates have a expiry set in the far future.